contracts/bounties/BreakInvariantBounty.sol

```diff
@@ -1,15 +1,15 @@
 pragma solidity ^0.4.24;
 
 
-import "./payment/PullPayment.sol";
-import "./lifecycle/Destructible.sol";
+import "../payment/PullPayment.sol";
+import "../lifecycle/Destructible.sol";
 
 
 /**
- * @title Bounty
+ * @title BreakInvariantBounty
  * @dev This bounty will pay out to a researcher if they break invariant logic of the contract.
  */
-contract Bounty is PullPayment, Destructible {
+contract BreakInvariantBounty is PullPayment, Destructible {
```

Comment:
Can you explain the reasoning behind this rename? I'm not sure I prefer the new name.

Reply:
Because now the directory is called bounties and it has room for other types of bounties, we need a less generic name. I used the docstring to come up with this name:
"This bounty will pay out to a researcher if they break invariant logic of the contract."

The alternative we had documented on the issue #1177 was to move it to examples, but we didn't document any other option for the name. Do you have a better name in mind, or would you like a different solution for taking it out of the root dir?

Reply:
Ah I remember that conversation now. The problem is that I can't imagine other types of bounties possible. Do you have any in mind?

Reply:
More or less. I might be stretching this a little, but there could be an oracle that checks who closed a github bug, and then let that person claim the bounty. Following the same idea, a bounty on ZeppelinOS might totally on chain, but not just for breaking a contract but for publishing a new version or something. I think we will see more things related to bounties in the future, and they can all fit in this directory. But these are just predictions, I don't have a strong opinion about this directory. This was jsut the best option I could come up with, but I'm happy moving the contract somewhere else, giving it a different name, or dropping it.

Reply:
That's a super cool idea, but if it's ever developed I don't see it being a part of OpenZeppelin because it would be a complex system.

Reply:
An `OracleBounty` sounds like a nice and straightforward usage example of an `Oracle` interface, once that's finalized (#971), I'd like to see such a contract. However, I don't think we'd also want to have an in-house `Oracle` implementation, so I'm not sure if this kind of thing would incentive that.

I'd keep the `bounties` directory, if only to see if any interesting bounty-related contributions come our way.